domain/src/main/java/org/oppia/domain/analytics/EventStorageHelper.kt

```diff
@@ -0,0 +1,97 @@
+package org.oppia.domain.analytics
+
+import androidx.lifecycle.LiveData
+import org.oppia.app.model.EventLog
+import org.oppia.app.model.EventLogs
+import org.oppia.data.persistence.PersistentCacheStore
+import org.oppia.util.data.AsyncResult
+import org.oppia.util.data.DataProviders
+import org.oppia.util.logging.Logger
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class EventStorageHelper @Inject constructor(
```

Comment:
Can we make this work for all types of logs rather than just event logs ? Since we need to store crash logs too

Reply:
Done.

Reply:
This is not working for crash logs

Reply:
Done, PTAL :)